src/layouts/home-assistant-main.ts

```diff
@@ -7,6 +7,7 @@ import { listenMediaQuery } from "../common/dom/media_query";
 import { toggleAttribute } from "../common/dom/toggle_attribute";
 import { computeRTLDirection } from "../common/util/compute_rtl";
 import "../components/ha-drawer";
+import "../components/ha-snowflakes";
```

Comment:
This should not have been merged like this. Putting extra imports in the hot path of the frontpage opening needs to be carefully considered. 

Slow phones already have a hard time opening our frontend as it is, and we need to make sure that we limit what is loaded to the bare minimum.

I put a fix here:
https://github.com/home-assistant/frontend/pull/28716